packages/web-components/src/wallet-connection/connectKeplr.js

```diff
@@ -0,0 +1,61 @@
+// @ts-check
+import { Registry } from '@cosmjs/proto-signing';
+import {
+  SigningStargateClient,
+  AminoTypes,
+  defaultRegistryTypes,
+  createBankAminoConverters,
+  createAuthzAminoConverters,
+} from '@cosmjs/stargate';
+import { Errors } from '../errors.js';
+import { agoricConverters, agoricRegistryTypes } from './signerOptions.js';
+
+/** @typedef {import('@keplr-wallet/types').Keplr} Keplr */
+
+/**
+ *
+ * @param {string} chainId
+ * @param {string} rpc
+ */
+export const connectKeplr = async (chainId, rpc) => {
```

Comment:
Maybe add `aminoTypes` and `registry` as arguments with default values? `accountParser` is another thing a developer may want exposed.

Reply:
This isn't exported in the package level, it's just a backward-compatible default in case `clientConfig` isn't provided. My intention is if someone's using a signer for non-agoric stuff also, they should provide their own signing client with the help of `agoricRegistryTypes`/`agoricConverters`. If there's any `accountParser` stuff specific to agoric that we're missing we should provide that here though, is there?

Reply:
Correct nothing specific from agoric wrt to account parser. But if someone is using this signer implementation to connect to other chains, like https://github.com/Agoric/wallet-app/issues/128, they might need one.

More in general, if I'm a library author and wrapping something that takes options, I like to expose those options to the consumer to give more them flexibility and the code a longer shelf life.